### ISSUE TITLE
[BUG] When using multiple tenants, the tenantId from the web hooks should be used

### ISSUE THREAD

**Issue by mariusdatakrag** [webfox/laravel-xero-oauth2#89]
When using multiple tenants, the tenantId from the web hooks should be used

See screenshot for the fix.

![image](https://github.com/webfox/laravel-xero-oauth2/assets/81690364/e225336d-9c4e-4021-9533-cdf45df72534)

```
    public function getResource()
    {
        if ($this->getEventCategory() === 'INVOICE') {
            return $this->accountingApi
                ->getInvoice($this->getTenantId(), $this->getResourceId())
                ->getInvoices()[0];
        }
        if ($this->getEventCategory() === 'CONTACT') {
            return $this->accountingApi
                ->getContact($this->getTenantId(), $this->getResourceId())
                ->getContacts()[0];
        }
    }
```

**Comment by hailwood** [comment]
Thanks @mariusdatakrag,
Well spotted!

Would you like to submit a PR so the fix is attributed to you?

**Comment by JamesFreeman** [comment]
I believe can be closed now @hailwood

**Comment by hailwood** [comment]
Will be fixed in the upcoming 6.0 release.
